src/components/home.tsx

```diff
@@ -10,11 +11,20 @@ const Home: FC = () => {
     </ListGroup.Item>
   ));
     
+  const editor = (
+    <ListGroup.Item key='template editor'>
+      <Link to={`/template_editor`}>
+      Template Editor
+      </Link>
+    </ListGroup.Item>
+  );
+
   return (
     <div>
       <h1>Choose a Document Type</h1>
       <ListGroup>
           {templates}
+          {editor}
```

Comment:
For production, we won't want the editor listed on the home page.  We don't want the average user to know about the editor. The user will have to explicitly change the url to access the editor (an Easter egg).